plugins/module_utils/job.py

```diff
@@ -328,6 +328,7 @@ def _get_job_status_str(job_id="*", owner="*", job_name="*"):
     Say '"'||'job_id'||'":"'||value('JOBID'||"."||ix)||'",'
     Say '"'||'job_name'||'":"'||value('JNAME'||"."||ix)||'",'
     Say '"'||'subsystem'||'":"'||value('ESYSID'||"."||ix)||'",'
+    Say '"'||'sysname'||'":"'||value('SYSNAME'||"."||ix)||'",'
```

Comment:
Would it be possible to switch `sysname` in output to `system`? Some other modules accept the output JSON without modification, so it may be worth outputting `system` for consistency between module responses

Reply:
Yes, that would be better for consistency.